docs/notebooks/expected_improvement.pct.py

```diff
@@ -201,6 +201,54 @@ def build_model(data):
     idx_best=arg_min_idx,
 )
 
+# %% [markdown]
+# ## Batch-sequential strategy
```

Comment:
is this notebook is getting a little long? remember we also have MES in here (at least we will do when we test it)

Reply:
Indeed it's long. I suggest to keep it this way for now, but once MES is here, we split it in half, so the first one only contains the basics and the second has MES and qEI, and compares the regrets of both with EI.